spec/imsc-hrm.html

```diff
@@ -557,12 +566,16 @@ <h2>Paint Text</h2>
       <p class='note'>In the case where a property is <em>prohibited</em> in a profile of [[IMSC]], the computed value of the property
       specified in [[ttml2]] can be used.</p>
 
-      <p class='note'>While one-to-one mapping between characters and typographical glyphs is generally the rule in some scripts,
-      e.g. latin script, it is the exception in others. For instance, in arabic script, a character can yield multiple glyphs
-      depending on its position in a word. The Hypothetical Render Model always assumes a one-to-one mapping, but reduces the
-      performance of the glyph buffer for scripts where one-to-one mapping is not the general rule (see GCpy below).</p>
+      <p class='note'>While one-to-one mapping between <a>characters</a> and glyphs (as defined in [[i18n-glossary]])
+      is common in some scripts (such as the Latin script), the actual relationship is often more complex.
+      Some scripts, such as Arabic, use different glyphs for a given <a>code point</a> depending on its position in a word.
+      Some scripts require combining marks or use a sequence of code points to form a glyph.
+      Cases exist where a given sequence of code points can have different glyph representations depending on context.
+      The Hypothetical Render Model assumes that code points have a one-to-one mapping to a glyph,
+      but accounts for the above complexity by reducing the performance of the glyph buffer
+      for scripts where a one-to-one mapping is not the general rule (see GCpy below).</p>
 
-      <p>For each <a>glyph</a> associated with a character in a <a>presented region</a> of <a>Intermediate Synchronic Document</a>
+      <p>For each <a>glyph</a> associated with a <a>character</a> in a <a>presented region</a> of <a>Intermediate Synchronic Document</a>
```

Comment:
It is really hard to leave this sentence alone, but, aside from this note, I'm going to on the theory that you just explained that you know that there might not be a separate glyph for a given character.